### PR TITLE
Update api-manager.xml template

### PR DESCRIPTION
**Pull request by ruwiniwj**
Add the following to the api-manager.xml template;
- Monetization implementation details
- Whitelisted URIs for API 3.0.0 UI

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json [hunk 1]
```diff
@@ -40,6 +40,9 @@
   "apim.key_manager.pool.max_idle": "100",
   "apim.key_manager.pool.init_idle_capacity": "50",
   "apim.key_manager.key_validation_handler_impl": "org.wso2.carbon.apimgt.keymgt.handlers.DefaultKeyValidationHandler",
+  "apim.monetization.publish_duration": "1",
```
Comment (tharindu1st) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json]: 1m or 1s

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json]: This is publish duration in days, which corresponds to config  `<PublishTimeDurationInDays>`.

Comment (tharindu1st) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json]: can't we not to maintain a new config for that and keep in different way

Comment (malinthaprasan) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json]: @tharindu1st example?

Comment (tharindu1st) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json]: value should be 1d then.and in unit-resolver.json should have a mapping

Comment (ruwiniwj) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json]: Fixed with e2aab61

**Pull request merged**